src/index.scss

```diff
@@ -84,3 +84,9 @@ form {
 .font-size-base {
   font-size: $font-size-base !important;
 }
+
+// TODO: Class override should be a contribution to Paragon if not already
```

Comment:
nit: might be worth adding the paragon version your upstream fix was released for future reference (v22.8.1).